tests/site-nav.js

```diff
@@ -75,33 +76,74 @@ describe('SiteNav', function() {
       $('body').width(1000);
     });
 
+    describe('initMobileMenu()', function() {
+      it('should append the mobile menu', function() {
+        expect(this.siteNav.$menu.find('.js-mobile-nav').length).to.equal(1);
+        expect(this.siteNav.isMobile).to.be.true;
+      });
+    });
+
     describe('assignAria()', function() {
       it('should assign aria attributes to the list and toggle', function() {
         expect(this.siteNav.$toggle.length).to.be.ok;
         expect(this.siteNav.$toggle.attr('aria-haspopup')).to.equal('true');
-        expect(this.siteNav.$list.attr('aria-hidden')).to.equal('true');
+        expect(this.siteNav.$menu.attr('aria-hidden')).to.equal('true');
       });
     });
 
     describe('toggle()', function() {
       function isOpen(siteNav) {
```

Comment:
Another strategy for this, you can name your function like `assertIsOpen()` and instead of returning a boolean, it can just throw an assertion error to fail the test. You could use `expect()` statements in here too which would improve the messaging in a test failure.